hhast-lint.json

```diff
@@ -24,6 +24,14 @@
       "extraLinters": [
         "Facebook\\HHAST\\DataProviderTypesLinter"
       ]
+    },
+    {
+      "patterns": ["src/Migrations/*"],
+      "linterConfigs": {
+        "Facebook\\HHAST\\HHClientLinter": {
+          "ignore": [5624, 5639]
```

Comment:
*bump* - this is what I'm referring to

Reply:
Currently the global ignore code list is `[5607, 5624, 5639]`, so this commit actually enabled 5607 error code in comparison to the global ignore list